src/components/ChannelsAvailability/ChannelsAvailability.tsx

```diff
@@ -230,11 +230,11 @@ const Channel: React.FC<ChannelProps> = ({
                           )
                         : ""
                     }
-                    value={publicationDate ? publicationDate : ""}
+                    value={publicationDate}
```

Comment:
If `null` is a valid value we should fallback to empty string `""` like before, otherwise we might get the infamous React warning "A component is changing an uncontrolled input..."

```suggestion
                    value={publicationDate ? publicationDate : ""}
```

Reply:
It looks like it was working fine without default `""`, but I can't find any fallback for that. I will change it anyway because otherwise it may have unexpected behavior in the future